src/main/java/uk/co/evoco/webdriver/configuration/WebDriverConfig.java

```diff
@@ -184,20 +172,39 @@ public void setRunType(String runType) {
     }
 
     /**
-     *
```

Comment:
These need adding back in - javadocs convention is to have a blank line first which is where the description goes